toolchains/compilers/gcc_arm_none_eabi/impl/gcc_arm_none_versions.bzl

```diff
@@ -51,8 +51,40 @@ _PLATFORM_SPECIFIC_CONFIGS_9 = {
     },
 }
 
+_WINDOWS_11 = {
+    "full_version": "11.3.rel1",
+    "remote_compiler": {
+        "url": "https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-mingw-w64-i686-arm-none-eabi.zip",
+        "sha256": "23f24595aa575fba4fdb0cb086df4b053862af60837502cb7e52bd4fb3d76c36",
+        "strip_prefix": "arm-gnu-toolchain-11.3.rel1-mingw-w64-i686-arm-none-eabi",
+    },
+}
+
+_PLATFORM_SPECIFIC_CONFIGS_11 = {
+    "linux": {
+        "full_version": "11.3.rel1",
+        "remote_compiler": {
+            "url":"https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi.tar.xz",
+            "sha256": "d420d87f68615d9163b99bbb62fe69e85132dc0a8cd69fca04e813597fe06121",
+            "strip_prefix": "arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi",
+        },
+    },
+    "windows": _WINDOWS_11,
+    "windows server 2019": _WINDOWS_11,
+    "windows 10": _WINDOWS_11,
+    "mac os x": {
+        "full_version": "11.3.rel1",
+        "remote_compiler": {
+            "url": "https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-darwin-x86_64-arm-none-eabi.tar.xz",
+            "sha256": "826353d45e7fbaa9b87c514e7c758a82f349cb7fc3fd949423687671539b29cf",
+            "strip_prefix": "arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi",
```

Comment:
I think this needs to be "arm-gnu-toolchain-11.3.rel1-darwin-x86_64-arm-none-eabi" the prefix is different for each platform.

See the build log here https://github.com/bazelembedded/bazel-embedded/actions/runs/3335309869/jobs/5519150813

Reply:
Hi
Thanks for the review, I have corrected the strip_prefix for darwin platform